README.md

```diff
@@ -99,89 +83,16 @@ Returns the record data serialized.
 ```js
 import * as ipns from 'ipns'
 
-const ipnsRecord = ipns.unmarshal(storedData)
+const ipnsRecord = ipns.unmarshalIPNSRecord(storedData)
 ```
 
 Returns the `IPNSRecord` after being deserialized.
 
-### Validator
```

Comment:
Have this and the other sections been removed because they are outdated? Are there replacements? We should update, instead of removing, if they contain valuable information not covered elsewhere.

Reply:
This is already covered by the validate record section. This removes the duplication.